modules/panorama/main.tf

```diff
@@ -15,14 +15,14 @@ resource "azurerm_network_interface" "this" {
   name                 = var.interface[0].name
   location             = var.location
   resource_group_name  = var.resource_group_name
-  enable_ip_forwarding = lookup(var.interface[0], "enable_ip_forwarding", "false")
+  enable_ip_forwarding = lookup(var.interface[0], "enable_ip_forwarding", false)
```

Comment:
Avoid `"false"` which has type `string`, to be compared to `false`, which has
type `bool`. This way if user uses `"False"` capitalized the behavior
will be saner.